script.js

```diff
@@ -0,0 +1,430 @@
+button.innerHTML = 'See Project';
+divWrap.appendChild(button);
+
+// -----------------------
+// loop to create the STANDARD cards
+// ----------------------
+for (let i = 1; i < 7; i += 1) {
+  // create STANDARD cards
+  article = document.createElement('article');
+  article.classList = `card card${i}`;
+  recentWorks.appendChild(article);
+
+  // create div
+  divWrap = document.createElement('div');
+  divWrap.classList = 'wrap-card';
+  article.appendChild(divWrap);
+
+  // create title of card
+  h3 = document.createElement('h3');
+  h3.classList = 'card-h';
+  h3.innerHTML = portifolio[i].heading;
+  divWrap.appendChild(h3);
+
+  // create p
+  p = document.createElement('p');
+  p.classList = 'card-p';
+  p.innerText = portifolio[i].highlight;
+  divWrap.appendChild(p);
+
+  // create div
+  div = document.createElement('div');
+  div.classList = 'lang';
+  divWrap.appendChild(div);
+
+  // create ul
+  ul = document.createElement('ul');
+  div.appendChild(ul);
+
+  // create li-items
+  li = document.createElement('li');
+  li.classList = 'lang-icons';
+  ul.appendChild(li);
+
+  // create badge items
+  li = document.createElement('li');
+  li.classList = 'lang-icons';
+  ul.appendChild(li);
+  badge = document.createElement('div');
+  badge.classList = 'badge';
+  badge.innerHTML = portifolio[i].badge1;
+  li.appendChild(badge);
+
+  li = document.createElement('li');
+  li.classList = 'lang-icons';
+  ul.appendChild(li);
+  badge = document.createElement('div');
+  badge.classList = 'badge';
+  badge.innerHTML = portifolio[i].badge2;
+  li.appendChild(badge);
+
+  li = document.createElement('li');
+  li.classList = 'lang-icons';
+  ul.appendChild(li);
+  badge = document.createElement('div');
+  badge.classList = 'badge';
+  badge.innerHTML = portifolio[i].badge3;
+  li.appendChild(badge);
+
+  // create the button see project
+  div = document.createElement('div');
+  div.classList = 'card-link';
+  article.appendChild(div);
+  button = document.createElement('button');
+  button.id = `see-project${i}`;
+  button.classList = 'see-project';
+  button.innerText = 'See Project';
+
+  div.appendChild(button);
+}
+// fim do loop
+
+// -------------------
+// create Details pop up
+// -------------------
+
+// // fill modal
+function fillModal(i) {
+  // create full screen container
+  const divContModal = document.createElement('div');
+  divContModal.classList = 'modal-container';
+  recentWorks.appendChild(divContModal);
+
+  // create modal
+  const divModal = document.createElement('div');
+  divModal.classList = 'details';
+  divContModal.appendChild(divModal);
+
+  // create x button
+  const xBtn = document.createElement('div');
+  xBtn.innerHTML = '<i id="xBtn" class="fa-solid fa-xmark"></i>';
+  divModal.appendChild(xBtn);
+
+  // create title of card
+  const h2 = document.createElement('h3');
+  h2.classList = 'details-title';
+  h2.innerHTML = portifolio[i].heading;
+  divModal.appendChild(h2);
```

Comment:
- [ ] the close button should be at right as in the Figma design with same color and also you should use the right font and  color for the title in order to match the design 
- [ ]  Yours
![image](https://user-images.githubusercontent.com/61790587/175617867-822012d0-f3d9-4e4f-aa65-8064fe0126e7.png)

- [ ] Figma
![image](https://user-images.githubusercontent.com/61790587/175617941-133d6449-3c6a-47b5-9cb9-3dc030a88dfb.png)